packages/hoppscotch-common/src/components/app/Banner.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    v-if="banner"
+    class="relative flex items-center px-4 py-2 transition text-tiny group"
+    :class="getBannerColor(banner.type)"
+  >
+    <icon-lucide-info class="mr-2 text-white" />
+    <span class="text-secondaryDark">
+      <span v-if="banner.alternateText" class="md:hidden">
+        {{ banner.alternateText(t) }}
```

Comment:
for the reasons mentioned in the next comment

```suggestion
        {{ banner.alternateText }}
```